src/components/SideNavbar/SideNavMenu.js

```diff
@@ -19,9 +19,8 @@ const Extension = () => {
           <S.ChatText>채팅방</S.ChatText>
         </S.ChatLink>
       </S.ChatList>
-
       <S.SetList>
-        <S.Set to="#">
+        <S.Set to="setting">
           <S.SetText>설정</S.SetText>
         </S.Set>

```

Comment:
```suggestion
        <S.Set to="mypage">
          <S.SetText>설정</S.SetText>
        </S.Set>

```
마이페이지부분으로 수정해주시면 감사하겠습니다.

Reply:
넵 수정하도록 하겠습니다!

Reply:
리뷰 감사합니다!

Reply:
[c17e721](https://github.com/BCSDLab/KOALA_FRONT_END/pull/47/commits/c17e7214702c738b41ad8011ffa880361528eca4)